vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -22,51 +34,62 @@
     @InjectMocks
     private ContentBlockManager managerToTest;
 
+    private ContentBlock contentBlock;
+    
     @Before
     public void setUp() {
         MockitoAnnotations.initMocks(this);
-
+        contentBlock = new ContentBlock();
+        contentBlock.setContentOrder(1);
     }
 
     @Test
     public void test_deleteTextBlockById_success() throws BlockDoesNotExistException {
         String textBlockId = "2";
-        managerToTest.deleteTextBlockById(textBlockId);
+        Optional<ContentBlock> ContentBlockOptional = Optional.of(contentBlock);
+        when(contentBlockRepository.findById("2")).thenReturn(ContentBlockOptional);
+        managerToTest.deleteTextBlockById(textBlockId,null);
```

Comment:
the test should get real data (not null) and also test the additional functionality (the logic you've added)